newIDE/app/src/MainFrame/index.js

```diff
@@ -449,6 +422,63 @@ const MainFrame = (props: Props) => {
     []
```

Comment:
It feels like to me it's self explanatory: if it's an empty list, it's equivalent to `componentDidMount`.
Is it okay for you?

Reply:
hmm, I'm still confused about the eslint above.
The goal of the message is for the next person to not remove the disable-eslint, because there's a message explaining why we're doing it only once (and why we shouldn't do it for every prop change)